android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/HomeContentDescription.kt

```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.home
+
+enum class HomeContentDescription(val value: String) {
+    TODAY_TEXT("TODAY_TEXT"),
+    CURRENT_DATE("CURRENT_DATE"),
+    AVATAR("AVATAR"),
+}
```

Comment:
Actually, we just need to have a const value for ui testing. We don't actually need the benefit of an enum. I think you could consider defining an object with constants instead. Technically constant saves more memory than enum https://stackoverflow.com/questions/25305523/enum-vs-static-constants-memory-footprint so use `enum` only when we need to extend its benefit 🙏 .

```
object HomeContentDescription {
    const val TODAY_TEXT = "TODAY_TEXT"
    const val CURRENT_DATE = "CURRENT_DATE"
    const val AVATAR = "AVATAR"
}
```

Reply:
Thanks for your explanation. I will update this.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/100/commits/5ad895b5ebe7f785e4dbbc060325c0509b5b89a7